src/sass_context_wrapper.cpp

```diff
@@ -1,3 +1,4 @@
+#include <stdlib.h>
```

Comment:
Please only add it in `src/sass_context_wrapper.h` and it will take care of the rest.